webpack.config.js

```diff
@@ -22,9 +22,8 @@ module.exports = {
   output: {
     filename: 'index.js',
     path: path.resolve(__dirname, 'dist'),
+    library: "RingBuffer",
```

Comment:
This wraps it in a new object as "namespace". So you must call "RingBuffer.RingBuffer" to reach the class. I would not do that. Anyway... I'll update the deps soon. Incl. tslint to eslint But thanks for your work.

Reply:
This is how I managed to get the ESM example in the original README working with the new Webpack version, I haven't tested the global script installation.